src/families/cosmos/types.js

```diff
@@ -161,3 +161,18 @@ export type TransactionRaw = {|
 |};
 
 export const reflect = (_declare: *) => {};
+
+type CosmosFormattedDelegationRaw = {
+  validator: ?CosmosValidatorItem,
+  address: string,
+  pendingRewards: BigNumber,
+  amount: BigNumber,
+  status: CosmosDelegationStatus,
+};
+
+export type CosmosFormattedDelegation =
+  | CosmosFormattedDelegationRaw
+  | (CosmosFormattedDelegationRaw & { formattedAmount: string })
+  | (CosmosFormattedDelegationRaw & { reward: string });
```

Comment:
this is a bit too complex type structure to me. what is the reason behind this? if we don't have a _field discriminator_ then it's probably not a good idea because you are opening a lot of problems with FlowType as it's not going to be able to differienciate.

**I would just add these as `?: string`**